naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java

```diff
@@ -298,7 +298,7 @@ public void run() {
                 updatedHealthStatus(namespaceId, serviceName, serverIP);
             } catch (Exception e) {
                 Loggers.SRV_LOG
-                        .warn("[DOMAIN-UPDATER] Exception while update service: {} from {}, error: {}", serviceName,
```

Comment:
Why to remove {}?

Reply:
placeholder {} let e.toString(), will lost error stackTrace details.